pkg/tracing/otlp/otlp.go

```diff
@@ -86,3 +92,20 @@ func newTraceProvider(ctx context.Context, processor tracesdk.SpanProcessor, log
 	)
 	return tp
 }
+
+func getSampler(config Config) (tracesdk.Sampler, error) {
+	switch strings.ToLower(config.SamplerType) {
+	case AlwaysSample:
+		return tracesdk.ParentBased(tracesdk.AlwaysSample()), nil
+	case NeverSample:
+		return tracesdk.ParentBased(tracesdk.NeverSample()), nil
+	case RatioBasedSample:
```

Comment:
As I pointed out above, this should be adjusted - for these samplers, we do **not** want to have them wrapped with `ParentBased`. We need to create a separate built-in samplers for those, so in total we should have 6 cases, something like:
- `AlwaysSample`
- `NeverSample`
- `TraceIDRatioBased`
- `ParentBasedAlwaysSample`
- `ParentBasedNeverSample`
- `ParentBasedTraceIDRatioBased`

Does that make sense?